src/pages/index.js

```diff
@@ -15,28 +15,26 @@ const featureRows = [
       imageUrlDark: 'img/community_dark.svg',
       link: 'docs/chain/collator',
       description: (
-        <>
-          Become a collator in for our Peregrine Testnet and later help
-          decentralizing the KILT Blockchain.
-        </>
+        <>Become a collator in for our Peregrine Testnet and later help decentralizing the KILT Blockchain.</>
       ),
     },
     {
       title: 'Workshop',
       imageUrl: 'img/expert_light.svg',
       imageUrlDark: 'img/expert_dark.svg',
       link: 'docs/sdk/workshop/welcome',
-      description: (
-        <>Explore the most fundamental features of the KILTprotocol.</>
-      ),
+      description: <>Explore the most fundamental features of the KILTprotocol.</>,
     },
     {
-      title: 'Whitepaper',
+      title: 'Whitepaper 2020',
       imageUrl: 'img/whitepaper_light.svg',
       imageUrlDark: 'img/whitepaper_dark.svg',
       link: 'https://www.kilt.io/wp-content/uploads/2020/01/KILT-White-Paper-v2020-Jan-15.pdf',
       description: (
-        <>Read up on the theoretical concepts behind KILT Protocol.</>
+        <>
+          Read the original whitepaper. KILT evolved a lot since then, but the ideas and values are still part of our
+          identity even though the technical details are outdated
```

Comment:
All the other descriptions end in a dot.
```suggestion
          identity even though the technical details are outdated.
```

Reply:
[84382bc](https://github.com/KILTprotocol/docs/pull/69/commits/84382bc6bf0872dfed55b7eabba5fe23b94cacbc) missed that one!